arithmetization/src/main/java/net/consensys/linea/zktracer/ZkTracer.java

```diff
@@ -64,25 +75,11 @@ public ZkTracer() {
 
   public ZkTracer(final LineaL1L2BridgeConfiguration bridgeConfiguration) {
     this.hub = new Hub(bridgeConfiguration.contract(), bridgeConfiguration.topic());
-
-    // Load opcodes configured in src/main/resources/opcodes.yml.
-    OpCodes.load();
-
-    // Load spillings configured in src/main/resources/spillings.toml.
-    try {
-      final TomlTable table =
-          Toml.parse(getClass().getClassLoader().getResourceAsStream("spillings.toml"))
-              .getTable("spillings");
-      table.toMap().keySet().forEach(k -> spillings.put(k, Math.toIntExact(table.getLong(k))));
-
-      for (Module m : this.hub.getModulesToCount()) {
-        if (!this.spillings.containsKey(m.moduleKey())) {
-          throw new IllegalStateException(
-              "Spilling for module " + m.moduleKey() + " not defined in spillings.toml");
-        }
+    for (Module m : this.hub.getModulesToCount()) {
+      if (!spillings.containsKey(m.moduleKey())) {
```

Comment:
can't we get the list of modules to count at Besu start up and verify once only? Instead of checking every time we instantiate the ZkTracer?

Reply:
you are right I misread the code